addons/survey/static/src/scss/survey_templates_results.scss

```diff
@@ -65,6 +65,9 @@
 }
 
 .o_survey_results_question {
+    .o_survey_results_question_badges {
```

Comment:
Instead of cancelling out the margin, could we instead define a rule that removes the margin-right for the last badge of the row?

Should be possible using the :last-child pseudo class normally.

(And yes, I still have a slight issue with negative margins ^^)

Side-note: If it lessens the diff and removes the extra indentation block, it's a bonus!

Reply:
@awa-odoo Hello, indeed, I thought about those negative margins :p Here we could have several lines (many tags or medium screens), hence my solution (the last child would not be the only element on the right). Today, I've also thought also about a custom row margin which could lessen the diff and prevent the block. We should just reduce it of that 0.25rem margin :

`margin-right: calc(-.5 * var(--bs-gutter-x) - 4px)` instead of the default `margin-right: calc(-.5 * var(--bs-gutter-x) )`

What do you think?

Cheers!